server/controllers/groups.js

```diff
@@ -69,10 +69,13 @@ export const getUsers = (req, res, next) => {
   const tiers = _.groupBy(req.group.tiers, 'name');
 
   const _isActive = (backer) => {
-    if (tiers[backer.tier] && tiers[backer.tier].interval === 'monthly' && now.diff(moment(backer.lastDonation), 'days') > 31)
-      return false
-    else
-      return true;
+    if (tiers[backer.tier]) {
+      if (tiers[backer.tier][0].interval === 'monthly' && now.diff(moment(backer.lastDonation), 'days') > 31)
+        return false;
+      if (now.diff(moment(backer.lastDonation), 'days') > 365)
```

Comment:
I tested it and Yahoo doesn't show up anymore.
It's not based on the donation but on the definition of the tier. Since the sponsor tier is defined as "monthly", Yahoo is not considered as a sponsor after 31 days.
For tiers defined as yearly or one-time, I consider a backer active for 365 days.